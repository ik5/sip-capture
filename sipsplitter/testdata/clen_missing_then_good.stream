INVITE foo@bar SIP/2.0

SIP/2.0 200 OK
Content-Length: 0

