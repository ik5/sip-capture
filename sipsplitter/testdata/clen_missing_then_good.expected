{
  "Original": "INVITE foo@bar SIP/2.0\r\n\r\nSIP/2.0 200 OK\r\nContent-Length: 0\r\n\r\n",
  "Discarded": 1,
  "NoStartLine": 0,
  "StartLine": 2,
  "NoHeaders": 0,
  "Header": 2,
  "NoBody": 0,
  "Body": 1,
  "Complete": 1,
  "Discards": [
    "INVITE foo@bar SIP/2.0\r\n\r\n"
  ],
  "Starts": [
    "INVITE foo@bar SIP/2.0\r\n",
    "SIP/2.0 200 OK\r\n"
  ],
  "Headers": [
    "",
    "Content-Length: 0\r\n"
  ],
  "Bodies": [
    ""
  ],
  "Messages": [
    "SIP/2.0 200 OK\r\nContent-Length: 0\r\n\r\n"
  ]
}